otherlibs/unix/unixsupport.h

```diff
@@ -43,18 +86,44 @@ extern void caml_unix_check_path(value path, const char * cmdname);
 
 #define DIR_Val(v) *((DIR **) &Field(v, 0))
 
+#ifdef _WIN32
+extern wchar_t ** cstringvect(value arg, char * cmdname);
+extern void cstringvect_free(wchar_t **);
+#else
 extern char ** cstringvect(value arg, char * cmdname);
 extern void cstringvect_free(char **);
+#endif
```

Comment:
It would be better if this used `char_os` instead:
```suggestion
extern char_os ** cstringvect(value arg, char * cmdname);
extern void cstringvect_free(char_os **);
```
however, that requires `caml/misc.h` and I'm not sure if there are any implications to adding that here? (I don't think there are??)